project/profiles/capacity4more/modules/c4m/content/c4m_content_wiki_page/c4m_content_wiki_page.features.field_base.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @file
+ * c4m_content_wiki_page.features.field_base.inc
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function c4m_content_wiki_page_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'field_c4m_editing_open'.
+  $field_bases['field_c4m_editing_open'] = array(
```

Comment:
@amitaibu - @ordavidil 
How would you do this? Create a realm per group (with the group id in the realm?) and use the user ids as GID?
And what is the use of this basefield?

If you are sure we can implement this for all these scenarios:

- A wiki page can be closed for modifications. (in current scope)
- A wiki page can be opened for ALL group members (automatically for new/removed members) to be edited (in current scope)
- A wiki page can be opened for specific group members (uid per uid) to be edited (in future scope)

Please go ahead with the solution you have experience with.